lambdas/cfn-list-transform-macro/list_transform/app.py

```diff
@@ -0,0 +1,32 @@
+import json
+import traceback
+
+
+def handle_transform(parameters):
+  operation = parameters['Operation']
+  transform_string = parameters['TransformString']
+  fragment = []
+  input_list = parameters['List']
+  prepend = parameters['Prepend']
+  if prepend:
+    fragment = [transform_string + item for item in input_list]
+  else:
+    fragment = [item + transform_string for item in input_list]
+  return fragment
+
+
+def handler(event, context):
+    status = 'success'
+    response = {
+      'requestId': event['requestId'],
+      'status': 'success'
+    }
+
+    try:
+      response['fragment'] = handle_transform(event['params'])
```

Comment:
I believe the parameters are in a `templateParameterValues` key, so this should be something like..
```
parameters = event["templateParameterValues"]
fragment = handle_transform(parameters)
```

Reply:
"params" is a sibling of "templateParameterValues" -- [reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-macros.html#template-macros-lambda-interface)